example_project/karma.conf.ts

```diff
@@ -122,20 +122,23 @@ const browsers = {
     osVersion: '12',
     browserName: 'Safari',
     useHttps: true,
+    flags: ['mobileuseragent'],
```

Comment:
This is a magic keyword, so for reliability, please import it from Broyster instead of typing here. For example:

```ts
import { BrowserFlag } from '@fpjs-incubator/broyster/node'

// ...

flags = [BrowserFlag.MobileUserAgent]
```

Reply:
- `BrowserFlags.Arguments.MobileUserAgent` looks weird. Please achieve what I suggested above. Tip: `export { ... as ... } from ...`.
- Add this new exported value to the documentation where you described the flags.

Reply:
my reasoning was that there are also firefox profile settings (which are a separate notion) that we could export in the future and this would provide a common entrypoint. did I overthink?

Reply:
Usually exports aren't grouped together in JavaScript, because it's bad for tree-shaking. I see it like this: `import { BrowserFlag, FirefoxArgument } from '@fpjs-incubator/broyster/node'`.

I suggested the name `BrowserFlag` for the enum because it's for browser configuration and the option name is `flags`.